team-fingerprints-web/src/components/OnboardingProfileModal/__tests__/OnboardingProfileModal.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import OnboardingProfileModal from "../OnboardingProfileModal";
+import { render, screen, fireEvent } from "@testing-library/react";
+import { AppSetup } from "testing/helpers/AppSetup";
+import userEvent from "@testing-library/user-event";
+import { createMemoryHistory, MemoryHistory } from "history";
+
+const setVisibleProfileModalMock = jest.fn();
+
+interface RenderComponentProps {
+  history?: MemoryHistory;
+  visibleProfileModal?: boolean;
+}
+
+const renderComponent = ({
+  history,
+  visibleProfileModal = true,
+}: RenderComponentProps) => {

```

Comment:
You can assign empty object here so you don't have to pass it every time you are rendering without any parameters. Makes it look cleaner 😄 
```suggestion
}: RenderComponentProps = {}) => {
```